static.config.js

```diff
@@ -725,7 +728,7 @@ const makeAllPages = async (langCode, incrementalPageId) => {
 
   let pages = [];
   let after = '';
-  let batchSize = Number(process.env.REACT_STATIC_BATCH_SIZE) || null;
+  let batchSize = Number(process.env.GRAPHQL_BATCH_SIZE) || null;
```

Comment:
I had plans for adding GRAPHQL_BATCH_SIZE to our Publisher variables. It's a better name. And it's confusing to have a var prefixed with REACT_STATIC_* when it isn't part of our library. If GRAPHQL_BATCH_SIZE never gets added to the Publisher, it'll be fine. Everything will still build with their default batchSize values.